bin/deploy

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+ENV=$1
+BRANCH_NAME="${BRANCH:-main}"
+REPOSITORY="${REPO:-dpul-collections}"
+JOB_NAME="${JOBNAME:-dpulc}"
+if [[ -z ${ENV} ]];
+then
+  echo "Missing Environment. Command: BRANCH=main ./bin/deploy staging"
+  exit
+fi
+# Notify Github of Starting Deploy, Deploy, Notify Github of Ending Deploy
+ssh -qtt deploy@nomad-host-prod1.lib.princeton.edu << EOF
```

Comment:
Add a comment: Run the github commands from the host machine because the pulbot token is set up there for the deploy user.